packages/amplify-category-interactions/provider-utils/awscloudformation/index.js

```diff
@@ -40,11 +40,26 @@ function copyCfnTemplate(context, category, options, cfnFilename) {
       target: `${targetDir}/${category}/${options.resourceName}/src/cfn-response.js`,
     },
   ];
-  Object.assign(defaultValues, options);
+  Object.assign(defaultValues, options, { mappings });
   // copy over the files
   return context.amplify.copyBatch(context, copyJobs, defaultValues, true, false);
 }
 
+function getTemplateMappings(context) {
+  const Mappings = {
+    RegionMapping: {},
+  };
+  const providerPlugins = context.amplify.getProviderPlugins(context);
+  const provider = require(providerPlugins['awscloudformation']);
+  const regionMapping = provider.getPinpointRegionMapping();
```

Comment:
this should be getLexRegionMapping, not?

Reply:
🤦